src/util/RouterUtil.js

```diff
@@ -233,19 +233,29 @@ fn.handleResponseStatus = function(router, res) {
   case 'PASSWORD_RESET':
     router.navigate('signin/password-reset', { trigger: true });
     return;
-  case 'LOCKED_OUT':
+  case 'LOCKED_OUT': {
     if (router.settings.get('features.selfServiceUnlock')) {
       router.navigate('signin/unlock', { trigger: true });
     } else {
-      router.controller.model.trigger('error', router.controller.model, {
+      const errorMessage = loc('error.auth.lockedOut', 'login');
+      const resError = {
         responseJSON: {
           errorCauses: [],
-          errorSummary: loc('error.auth.lockedOut', 'login'),
+          errorSummary: errorMessage,
+          errorCode: 'E0000069',
```

Comment:
Are we receiving that errorCode from the backend response? Instead of hardcoding it here, can we pass along whatever we received from the backend?